roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -861,3 +892,58 @@ resources:
           floating_network: {{ openshift_openstack_external_network_name }}
 {% endif %}
           volume_size: {{ openshift_openstack_cns_volume_size }}
+
+
+{% if openshift_openstack_use_lbaas_load_balancer %}
+  # NOTE: we're using the same LB for both API and Routers
+  loadbalancer:
+    type: OS::Neutron::LBaaS::LoadBalancer
+    properties:
+      vip_subnet: { get_resource: subnet }
+
+
+  lb_floating_ip:
+    condition: { not: no_floating }
+    depends_on:
+    - loadbalancer
+    - api_listener
+    - api_lb_pool
+    - router_listener
+    - router_lb_pool
+    type: OS::Neutron::FloatingIP
+    properties:
+      floating_network: {{ openshift_openstack_external_network_name }}
+      port_id: { get_attr: [loadbalancer, vip_port_id] }
+
+
+  api_listener:
+    type: OS::Neutron::LBaaS::Listener
+    properties:
+      protocol: HTTPS
+      # TODO(shadower): change this to 443?
+      protocol_port: 8443
+      loadbalancer: { get_resource: loadbalancer }
+
+  api_lb_pool:
+    type: OS::Neutron::LBaaS::Pool
+    properties:
+      lb_algorithm: ROUND_ROBIN
```

Comment:
nit: we prolly want this configurable

Reply:
The previous default (the VM-based LB) was not configurable either, so I'm tempted to leave this as is for now. But I've added a TODO to make it configurable in the future.